src/network/netdev/tuntap.c

```diff
@@ -241,6 +227,28 @@ static int tuntap_verify(NetDev *netdev, const char *filename) {
                                    "Please set it in the corresponding .network file.",
                                    netdev_kind_to_string(netdev->kind), filename);
 
+        if (t->user_name) {
+                const char *user = t->user_name;
+                uid_t uid;
+
+                r = get_user_creds(&user, &uid, NULL, NULL, NULL, USER_CREDS_ALLOW_MISSING);
```

Comment:
maybe prefix the 3rd, 4th, 5th parameter with `/* ret_gid= */` style comments, as usual, if you touch this?